plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTypeUtil.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake;
+
+import io.trino.plugin.hive.HiveType;
+import io.trino.plugin.hive.HiveTypeName;
+import io.trino.spi.type.Type;
+import org.apache.hadoop.hive.ql.io.IOConstants;
+
+import java.util.List;
+import java.util.Properties;
+
+import static java.util.stream.Collectors.joining;
+
+public final class DeltaLakeTypeUtil
+{
+    private DeltaLakeTypeUtil() {}
+
+    public static Properties buildSchemaProperties(List<String> columnNames, List<Type> columnTypes)
```

Comment:
Note that there exists already a utility class called `DeltaLakeSchemaProperties`.

There exists in `trino-hive` also a similar utility method called `io.trino.plugin.hive.metastore.MetastoreUtil#getHiveSchema(io.trino.plugin.hive.metastore.Table)`


What about using similarly a name like `buildHiveSchema` ?

Reply:
Thanks, I was missing the class since it was not final.